otter/convergence/steps.py

```diff
@@ -70,12 +102,38 @@ def got_name(random_name):
                 'POST',
                 'servers',
                 data=thaw(server_config),
-                success_pred=has_code(202))
+                success_pred=has_code(202),
+                reauth_codes=(401,))
 
         def report_success(result):
             return StepResult.SUCCESS, []
 
         def report_failure(result):
+            """
```

Comment:
I think the parsing should be in separate function: `parse_nova_api_error` that takes `APIError` and returns `message`. We soon need to decide about how to transfer structured info from the error.